simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -12,6 +12,9 @@
 
     private static final int DEFAULT_HEARTBEAT_INTERVAL = 60;
 
+    private static final long DEFAULT_METER_VALUES_INTERVAL = 1_000;
+    private static final long DEFAULT_POWER_CONSUMPTION_PER_INTERVAL = 100;
```

Comment:
We can mention that this is 'Wh' unit (since we don't set any unit to MeterValuesType, so watt-hour is a default unit according to OCPP spec).
DEFAULT_CONSUMPTION_WATT_HOUR